partials/byline-multiple.hbs

```diff
@@ -12,9 +12,17 @@
                 <div class="author-card">
                     <div class="basic-info">
                         {{#if profile_image}}
-                            <img class="author-profile-image" src="{{img_url profile_image size="avatar_medium_2x"}}" alt="{{name}}" />
+                            <img
+                                class="author-profile-image"
+                                src="{{img_url profile_image size="avatar_medium_2x"}}"
+                                alt="{{name}}"
```

Comment:
Nestálo by za zvážení sem přidat atributy `width` a `height`, nebo se tam generují automaticky?

Reply:
Obdobně na řádku 41.

Reply:
Velikost neudávám, protože v CSS je fluidní – svázaná s `vw`. Mohl bych zkusit dát sem 70 px, což je nějaká střední hodnota. Ale při načítání kvůli tomu stránka neposkakuje, tak bych to klidně nechal.